.github/workflows/rerun.yml

```diff
@@ -0,0 +1,25 @@
+# Cross version tests sometimes fail due to transient errors. This workflow reruns failed tests.
+name: Rerun
+
+on:
+  schedule:
+    # Run this workflow daily at 17:00 UTC (4 hours after cross-version-tests.yml workflow)
+    - cron: "0 17 * * *"
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event_name }}-${{ github.head_ref || github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  set-matrix:
+    runs-on: ubuntu-latest
+    timeout-minutes: 10
+    if: github.repository == 'mlflow/mlflow'
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/github-script@v6
+        with:
+          github-token: ${{ secrets.MLFLOW_AUTOMATION_TOKEN }}
+          script: |
+            const rerun = require(`${process.env.GITHUB_WORKSPACE}/.github/workflows/rerun.js`);
+            awiat rerun({ context, github, workflow_id: "cross-version-tests.yml" });
```

Comment:
typo? awiat => await

I am wondering why this is not captured by the test..

Reply:
great catch!

Reply:
> I am wondering why this is not captured by the test..

because it's not executed. This is a cron job, not executed on RPs.

Reply:
ah makes sense!